alerts.css

```diff
@@ -0,0 +1,60 @@
+.container-alerts {
+  display: flex !important;

```

Comment:
Try to avoid !important in the code , as mention by Vikram its not a good practice, and in your situation I dont think you need !important , because the css is anyway going to override, let me know if yu need help on this .

Else code looks good to merge